python/wpt/exporter/step.py

```diff
@@ -78,6 +79,9 @@ def run(self, run: SyncRun):
         try:
             commits = self._get_upstreamable_commits_from_local_servo_repo(run.sync)
             branch_name = self._create_or_update_branch_for_pr(run, commits)
```

Comment:
Looking at the code for `_create_or_update_branch_for_pr`, it looks like we can annotate it to return `str` always. I believe the `finally` clause at the end can also return `branch_name` based on the logic there, after the try/except that deletes the branch. This way, we don't need the `is None` check below.

`wpt_branch_name_from_servo_pr_number` in the common.py can also be annotated to return `str`